--- a/LSL/liblsl/lslboost/libs/thread/src/win32/thread.cpp
+++ b/LSL/liblsl/lslboost/libs/thread/src/win32/thread.cpp
@@ -637,9 +637,7 @@ namespace lslboost
                 timer_handle=CreateWaitableTimer(NULL,false,NULL);
                 if(timer_handle!=0)
                 {
-                    ULONG tolerable=32; // Empirical testing shows Windows ignores this when <= 26
-                    if(time_left_msec/20>tolerable)  // 5%
-                        tolerable=static_cast<ULONG>(time_left_msec/20);
+                    ULONG tolerable=1; // Hopefully this works.
                     LARGE_INTEGER due_time={{0,0}};
                     if(time_left_msec>0)
                     {
@@ -729,9 +727,7 @@ namespace lslboost
                 timer_handle=CreateWaitableTimer(NULL,false,NULL);
                 if(timer_handle!=0)
                 {
-                    ULONG tolerable=32; // Empirical testing shows Windows ignores this when <= 26
-                    if(time_left_msec/20>tolerable)  // 5%
-                        tolerable=static_cast<ULONG>(time_left_msec/20);
+                    ULONG tolerable=1; // Hopefully this works.
                     LARGE_INTEGER due_time={{0,0}};
                     if(time_left_msec>0)
                     {
